addons/sale_project/views/project_update_template.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
 
     <template id="sale_project.milestone_deadline_inherit" inherit_id="project.milestone_deadline">
-        <xpath expr="//t[contains(@t-if, 'deadline')]" position="before">
+        <xpath expr="//t[@t-if=&quot;milestone['deadline']&quot;]" position="before">
```

Comment:
That commit should be in your first commit since the previous change was done in your first commit